src/main/java/formflow/library/data/Submission.java

```diff
@@ -97,7 +97,7 @@ public Submission() {
    */
   public Map<String, Object> getSubflowEntryByUuid(String subflowName, String uuid) {
     if (inputData.containsKey(subflowName)) {
-      ArrayList<Map<String, Object>> subflow = (ArrayList<Map<String, Object>>) inputData.get(subflowName);
+      List<Map<String, Object>> subflow = (List<Map<String, Object>>) inputData.get(subflowName);
```

Comment:
Was getting an error when trying to cast Immutable List to ArrayList

Reply:
That's strange that we haven't gotten that before.   It's probably better being a List anyways.